src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/GenericConstraintTests.vb

```diff
@@ -3889,12 +3889,6 @@ End Class
                 </compilation>
             Dim compilation = CreateCompilationWithCustomILSource(vbSource, ilSource)
             compilation.AssertTheseDiagnostics(<errors><![CDATA[
```

Comment:
>AssertTheseDiagnostics [](start = 24, length = 22)

It feels like this change of behavior in VB (between new compiler and old compiler) may set up compat issues for customers, even though they are compiling with VB LangVersion 15.
Something to consider as part of the follow-up issue.

Reply:
> Something to consider as part of the follow-up issue.

I will capture that too.

---
In reply to: [217167443](https://github.com/dotnet/roslyn/pull/29809#discussion_r217167443) [](ancestors = 217167443)